shared/package.json

```diff
@@ -166,11 +166,12 @@
     "redux-logger": "3.0.6",
     "source-map-support": "0.4.15",
     "style-loader": "0.18.1",
+    "uglify-js": "git://github.com/mishoo/UglifyJS2#harmony-v2.8.22",
```

Comment:
this is really uglify-es but it allows us to use the uglifyfs-webpack-plugin w/ es6 support